### PR TITLE
Automatic timer: Only activate for players

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

**Comment by DaWoblefet** [comment]
thankyouthankyouthankyouthankyouthankyouthankyouthankyouthankyouthankyouthankyou

**Comment by Zarel** [comment]
This is gonna conflict with my code :s

File: js/client-battle.js [hunk 1]
```diff
@@ -47,7 +47,11 @@
 			this.battle.startCallback = function () { self.updateControls(); };
 			this.battle.stagnateCallback = function () { self.updateControls(); };
 
-			if (Dex.prefs('autotimer')) this.setTimer('on');
+			if (Dex.prefs('autotimer')) {
+				var user = this.users[app.user.get('userid')];
+				if (user && user.group === '☆') this.setTimer('on');
```
Comment (Zarel) [inline js/client-battle.js:52]: In the meantime, can we escape this?

Comment (AnnikaCodes) [reply js/client-battle.js]: sure

**Comment by Zarel** [comment]
#1686 isn't going to get merged anytime soon so this gets precedence.

**Pull request merged**